src/cordova/platform/addHelper.js

```diff
@@ -99,9 +99,9 @@ function addHelper (cmd, hooksRunner, projectRoot, targets, opts) {
                     }
 
                     // If spec still doesn't exist, try to use pinned version
```

Comment:
nitpik: Comment still states attempting to use pinned version.

Reply:
I updated the comment.